### ISSUE TITLE
&rsquo; rendered on the homepage snippets

### ISSUE THREAD

**Issue by morariu** [wangchucheng/hugo-eureka#17]
`&rsquo;` rendered on the homepage snippets. On the individual pages, the text renders just fine. This is on my localhost.

`Politicians don&rsquo;t understand`

**Comment by morariu** [comment]
Hugo Static Site Generator v0.78.2/extended linux/amd64 BuildDate: 2020-11-13T14:10:43Z
GOOS="linux"
GOARCH="amd64"
GOVERSION="go1.15.4"

**Comment by wangchucheng** [comment]
@morariu Hi, I've tested it in my machine, and it works well.

![image](https://user-images.githubusercontent.com/38368052/101655410-8cf2a500-3a7c-11eb-86f5-8dcbccd8229d.png)

I just pasted `Politicians don&rsquo;t understand`, and my hugo version is 0.78.2. I have not tested it in extended version, but I guess this is irrelevant with that.

If there is anything else that can help you, I am willing to help.
